assets/less/zenodo-rdm/globals/site.overrides

```diff
@@ -164,3 +164,14 @@ a.inverted {
 .cookie-banner.hidden {
   display: none !important;
 }
+
+.preview-iframe {
+  height:60vh;
```

Comment:
Maybe to be tested on a few different viewports with a few different file types (e.g. zip) and validate that the UX is fine.